src/main/java/bio/terra/app/configuration/UserMetricsConfiguration.java

```diff
@@ -18,6 +18,8 @@ public class UserMetricsConfiguration {
   private String appId;
   private String bardBasePath;
   private Integer metricsReportingPoolSize;
+
+  private Integer syncRefreshInterval;
```

Comment:
It's hard to tell from the parameter name if this is seconds/minutes, etc.  I'm assuming it's seconds?  A few of our parameters have `Seconds` at the end for this reason.  Probably worth using that convention here